lms/lti.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass
+from typing import Callable, Optional
+
+
```

Comment:
This is in a new top level `lti.py` file. It could go in a different place but not fit any of them perfectly.

This being just plain python (no pyramid, no marshmallow, etc) I think is fine to leave it at the top level.